Formula/cbmc.rb

```diff
@@ -0,0 +1,53 @@
+class Cbmc < Formula
+  desc "CBMC: The C Bounded Model Checker"
+  homepage "https://www.cprover.org/cbmc/"
+  url "https://github.com/diffblue/cbmc.git",
+      :using    => :git,
+      :tag      => "cbmc-5.12",
+      :revision => "d51243d346350127ecb269f1e546cbd850ef6955"
+  sha256 "1b9d003e1baffc714b36a38087e4ed42b47c04da5ebdb02bbce03262ea3acafd"
+
+  depends_on "cmake" => :build
+  depends_on "maven" => :build
+  depends_on "ninja" => :build
+  depends_on "openjdk" => :build
+
```

Comment:
None of these seem to be used. Are you sure they're all required?

Reply:
I've removed ninja.  (I deleted -GNinja from cmake by mistake, but I'll leave it off.)

Building jbmc requires jdk and maven:  "To compile JBMC, you additionally need the JDK and Maven 3." is from https://github.com/diffblue/cbmc/blob/develop/COMPILING.md